wallet-test/src/test/scala/org/bitcoins/wallet/UTXOLifeCycleTest.scala

```diff
@@ -32,20 +32,25 @@ class UTXOLifeCycleTest extends BitcoinSWalletTest {
     val WalletWithBitcoindRpc(wallet, _) = param
 
     for {
+      oldTransactions <- wallet.listTransactions()
       tx <- wallet.sendToAddress(testAddr,
                                  Satoshis(3000),
                                  Some(SatoshisPerByte(Satoshis(3))))
 
       updatedCoins <- wallet.spendingInfoDAO.findOutputsBeingSpent(tx)
+      newTransactions <- wallet.listTransactions()
     } yield {
       assert(updatedCoins.forall(_.state == TxoState.PendingConfirmationsSpent))
+      assert(!oldTransactions.map(_.transaction).contains(tx))
```

Comment:
nit: maybe use `.forall()` here? and everywhere else this pattern is repeated in the file